### PR TITLE
my great contribution

### PR DESCRIPTION
**Pull request by StevenClontz**
This is what we discussed to make an awesome contribution during our Oct 10 meetup.

### CODE REVIEW

**Comment by siwelwerd** [comment]
I don't think we really want to include this at this point.

**Review by siwelwerd — CHANGES_REQUESTED** [review]
Still needs some work!

File: source/03-PR/02.ptx [hunk 1]
```diff
@@ -13,16 +13,36 @@
     <title>Activities</title>
     
     <p>Activities go here! Don't forget to put text in <c>&lt;p&gt;</c> tags or it won't show up.</p>
+    <p>That's right.</p>
```
Comment (siwelwerd) [inline source/03-PR/02.ptx:16]: Do we really need this?

**Comment by StevenClontz** [comment]
Actually, this was just a demo, so let's close this.

**Pull request closed without merging**